From: gregkh@suse.de
Subject: [PATCH] USB: give sisusb a valid minor number (133 - 140)
ChangeSet 1.2020.1.4, 2005/03/07 22:17:17-08:00, gregkh@suse.de

[PATCH] USB: give sisusb a valid minor number (133 - 140)

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


 drivers/usb/misc/sisusbvga/sisusb.h |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)


diff -Nru a/drivers/usb/misc/sisusbvga/sisusb.h b/drivers/usb/misc/sisusbvga/sisusb.h
--- a/drivers/usb/misc/sisusbvga/sisusb.h	2005-03-08 16:56:32 -08:00
+++ b/drivers/usb/misc/sisusbvga/sisusb.h	2005-03-08 16:56:32 -08:00
@@ -54,7 +54,7 @@
 
 /* USB related */
 
-#define SISUSB_MINOR	64		/* FIXME */
+#define SISUSB_MINOR	133		/* FIXME */
 
 /* Size of the sisusb input/output buffers */
 #define SISUSB_IBUF_SIZE  0x01000
